src/app/portlets/content-types/edit/content-types-edit.component.ts

```diff
@@ -217,13 +217,13 @@ export class ContentTypesEditComponent implements OnInit, OnDestroy {
     removeFields(fieldsToDelete: ContentTypeField[]): void {
         this.fieldService
             .deleteFields(this.data.id, fieldsToDelete)
-            .pipe(pluck('fields'))
+            .pipe(pluck('fields'), takeUntil(this.destroy$))
```

Comment:
`take(1)`? this happen everytime we remove a field so we're doing subs everytime we call this method, so the `take(1)` will do the sub, get the first response and completes, I think that's what we need right?

Reply:
?

Reply:
?